src/lib/hash/sha2_32/sha2_32.cpp

```diff
@@ -68,95 +68,84 @@ void SHA_256::compress_digest(digest_type& digest, std::span<const uint8_t> inpu
    uint32_t A = digest[0], B = digest[1], C = digest[2], D = digest[3], E = digest[4], F = digest[5], G = digest[6],
             H = digest[7];
 
+   std::array<uint32_t, 16> W;
+
    BufferSlicer in(input);
 
    for(size_t i = 0; i != blocks; ++i) {
```

Comment:
There is also the BMI2 SHA-2 which should probably be updated for parity

Reply:
Thanks. Done.